src/include/gurt/telemetry_consumer.h

```diff
@@ -1,5 +1,5 @@
 /**
- * (C) Copyright 2020 Intel Corporation.
+ * (C) Copyright 2020-2021 Intel Corporation.
```

Comment:
Just noticed the contract number in the copyright block isn't current. Should be 8F-30005. Same applies to telemetry_producer.h

Reply:
Fixed.